build.gradle

```diff
@@ -123,6 +123,7 @@ dependencies {
 
   compile group: 'io.github.openfeign', name: 'feign-httpclient', version: '9.5.0'
   compile group: 'uk.gov.hmcts.reform', name: 'document-management-client', version: '1.0.2'
+  compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '0.0.4'
```

Comment:
this is missing a heathcheck, preferably the library would provide a healthindicator and all consumers would automatically get it

Reply:
is it something new requirement, none of client I feel provide health indicator

Reply:
pdf-service-client provides a health method and in the claim-store we have a health indicator, that's the minimum needed.
but it would be much better if the client provides the health indicator.

This was missed on the new clients but it is a requirement that we have a healthcheck for each service we talk to.

Reply:
Good call @timja-kainos 👍